examples/arithmetics/README.md

```diff
@@ -0,0 +1,13 @@
+# Arithmetics Example
+
+## Interpreter CLI
+
+The Arithmetics Example features an interpreter that you can run via cli.
+
+Ensure the complete projects was properly built, otherwise run `npm install` from the root of the Langium project.
+Use `node ./bin/cli.js` to run the cli. Follow the instructions or use `node ./bin/cli.js eval <full-path-to-calc-file>`.
+
+## VSCode Extension
+
+Please see use the VSCode run configuration "Run Arithmetics Extension" to launch a new VSCode instance including the extension for this language.
```

Comment:
```suggestion
Please use the VSCode run configuration "Run Arithmetics Extension" to launch a new VSCode instance including the extension for this language.
```

Reply:
Yes, I createdI will change it accordingly